src/mavedb/lib/score_sets.py

```diff
@@ -56,7 +57,12 @@ def search_score_sets(db: Session, owner: Optional[User], search: ScoreSetsSearc
     # .filter(ScoreSet.private.is_(False))
 
     if owner is not None:
-        query = query.filter(ScoreSet.created_by_id == owner.id)
+        query = query.filter(
+            or_(
+                ScoreSet.created_by_id == owner.id,
+                ScoreSet.contributors.has(Contributor.orcid_id == owner.username),
```

Comment:
```suggestion
                ScoreSet.contributors.any(Contributor.orcid_id == owner.username),
```
Have to use `.any` on association proxy objects like in `lib/experiments.py`, or this will raise
> `sqlalchemy.exc.InvalidRequestError: 'has()' not implemented for collections. Use any().`

Reply:
Looks like I changed this at some point before the rebase.